src/main/java/org/reaktivity/nukleus/mqtt/internal/stream/MqttServerFactory.java

```diff
@@ -605,41 +613,43 @@ private int decodePublish(
         final int limit)
     {
         final int length = limit - offset;
-
         int progress = offset;
 
-        decode:
```

Comment:
This `decode:` label is in place to support single return while allowing an easy escape from the outer if block.
Please restore it and use `break decode` in place of `return` mid method.